keccak-hash/Cargo.toml

```diff
@@ -6,10 +6,15 @@ authors = ["Parity Technologies <admin@parity.io>"]
 repository = "https://github.com/paritytech/parity-common"
 readme = "README.md"
 license = "GPL-3.0"
+edition = "2018"
 
 [dependencies]
 tiny-keccak = "1.4"
-primitive-types = { path = "../primitive-types", version = "0.4" }
 
 [dev-dependencies]
 tempdir = "0.3"
+rustc-hex = { version = "2.0", default-features = false }
```

Comment:
Please revert this in favour of `hex_literal`. We are trying to slowly phase out `rustc-hex` and it should not be used by new code.

Reply:
Okay